### PR TITLE
[Complete] - Solution - Phase2 - Day3

### PR DESCRIPTION
**Pull request by haricharanbole**
Solved 

> **rot-13**

 problem by using for-of loop on ciphered string and some string methods to work with charcodes.

### CODE REVIEW

File: js-exercises/rot-13/rot13.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const rot13 = (cipheredText) => {
+  if (!isString(cipheredText)) {
+    throw new Error(
+      "Invalid Input Received. Exprected String. But, received " +
```
Comment (ahhda) [inline js-exercises/rot-13/rot13.js:4]: `Exprected` -> `Expected`

File: js-exercises/rot-13/rot13.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  }
+  return decipheredText;
+};
+
+const isString = (input) => {
+  return typeof input === "string";
+};
+
+const isCapitalAlphabet = (character) => {
+  return !(character == character.toLowerCase());
```
Comment (ahhda) [inline js-exercises/rot-13/rot13.js:24]: Why did you chose to use the loose equality here?

**Review by ahhda — CHANGES_REQUESTED** [review]

File: js-exercises/rot-13/rot13.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const rot13 = (cipheredText) => {
+  if (!isString(cipheredText)) {
+    throw new Error(
+      "Invalid Input Received. Exprected String. But, received " +
+        typeof cipheredText
```
[the remark this answers is not in the document]
Comment (haricharanbole) [reply js-exercises/rot-13/rot13.js]: Agree. I've updated.

File: js-exercises/rot-13/rot13.js (hunk 2 above)
Comment (haricharanbole) [reply js-exercises/rot-13/rot13.js]: I thought it's sufficient. Strict equality makes more sense. I've updated.

**Comment by haricharanbole** [comment]
I've pushed latest changes.

**Comment by haricharanbole** [comment]
Reviewed by Anuj. And, I've changed the requested changes.

**Pull request merged**